test/e2e-tests.sh

```diff
@@ -109,6 +109,14 @@ function dump_extra_cluster_state() {
   kubectl logs $(get_app_pod controller knative-serving)
 }
 
+function publish_test_images() {
+  image_dirs="$(find test/test_images -depth 1 -type d)"
```

Comment:
Shouldn't it be `maxdepth` here?

Reply:
`maxdepth` will include the current path (which is `test/test_images`).  `ko` will report error because it doesn't has a go package inside.